app/api/uploadthing/route.ts

```diff
@@ -1,7 +1,14 @@
 import { createRouteHandler } from 'uploadthing/next';
+import { env } from '~/env';
 import { ourFileRouter } from './core';
 
 // Export routes for Next App Router
 export const { GET, POST } = createRouteHandler({
   router: ourFileRouter,
+  config: {
+    callbackUrl:
+      env.NODE_ENV === 'production' && env.PUBLIC_URL
```

Comment:
Why is the check for production environment needed here (and if it is needed, why isn't it also used in the route change above)?

Again, would rather see nullish coalescing than ternary here.